support/k8s/k8s-workload-registrar/README.md

```diff
@@ -1,8 +1,13 @@
 # SPIRE Kubernetes Workload Registrar
 
 The SPIRE Kubernetes Workload Registrar implements a Kubernetes
-ValidatingAdmissionWebhook that facilitates automatic workload registration
-within Kubernetes.
+ValidatingAdmissionWebhook or informer that facilitates automatic
+workload registration within Kubernetes.
+
+In webhook mode, this tool is an http server which needs to be
```

Comment:
Sounds reasonable. Flag, or config file entry?

Reply:
(And I'll just note for future reference that the ways to implement a CRD mode are either to further extend this informer implementation, or switch to the far more complicated https://github.com/kubernetes-sigs/controller-runtime)

Reply:
I've made a first cut of this based around keeping the "mode" setting in the config file, so that it's self-contained. I can always move this to a flag if you think that's a good idea.

Reply:
Now moved the mode to a flag.